### ISSUE TITLE
[a11y]: Menu item in Dropdown component is not read by JAWS when it's focused

### ISSUE THREAD

**Issue by tshimizu-ibm** [carbon-design-system/carbon#19214]
### Package

@carbon/react

### Browser

Chrome

### Operating System

Windows

### Package version

1.80.1

### React version

18.3.1

### Automated testing tool and ruleset

N/A

### Assistive technology

JAWS

### Description

When focusing on a menu item in Dropdown menu by up/down arrow key, JAWS does not speak the item.
User cannot understand what item they are focusing right now unless they select it and close the menu by space/enter key.

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-dropdown--default

### Steps to reproduce

1. activate JAWS 2024
2. focus on a Dropdown box by tab key
3. press space key to open the menu
4. press down arrow key to move the focus to a menu item
5. JAWS does not speak the focused item in the menu

https://github.com/user-attachments/assets/b49f4130-f5d8-4c15-9af8-d1939635f013

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by wkeese** [comment]
Is this the same as https://github.com/carbon-design-system/carbon/issues/17582?

**Comment by tshimizu-ibm** [comment]
@wkeese Thanks, it seems so.
I checked https://react.carbondesignsystem.com/?path=/story/components-dropdown--default again and now it's `1.81.0` and verified the issue is fixed. JAWS reads all focused menu items.
